Client/Frontend/TabContentsScripts/CreditCardHelper.swift

```diff
@@ -156,4 +152,32 @@ class CreditCardHelper: TabContentScript {
 
         return injectionJSON
     }
+
+    // MARK: Next & Previous focus fields
+
+    static func focusNextInputField(tabWebView: WKWebView,
+                                    logger: Logger = DefaultLogger.shared) {
+        let fxWindowValExtras = "window.__firefox__.CreditCardExtras"
+        let fillCreditCardInfoCallback = "\(fxWindowValExtras).focusNextInputField()"
+
+        tabWebView.evaluateJavascriptInDefaultContentWorld(fillCreditCardInfoCallback) { _, err in
+            guard let err = err else { return }
+            logger.log("Unable to go next field: \(err)",
+                       level: .debug,
+                       category: .webview)
+        }
+    }
+
+    static func focusPreviousInputField(tabWebView: WKWebView,
+                                        logger: Logger = DefaultLogger.shared) {
+        let fxWindowValExtras = "window.__firefox__.CreditCardExtras"
+        let fillCreditCardInfoCallback = "\(fxWindowValExtras).focusPreviousInputField()"
+
+        tabWebView.evaluateJavascriptInDefaultContentWorld(fillCreditCardInfoCallback) { _, err in
+            guard let err = err else { return }
+            logger.log("Unable to go next field: \(err)",
```

Comment:
Nit: I think it should be previous field in this case
```suggestion
            logger.log("Unable to go previous field: \(err)",
```

Reply:
good catch 😅